yarn.lock

```diff
@@ -11887,11 +11902,16 @@
   resolved "https://registry.yarnpkg.com/@types/minimatch/-/minimatch-2.0.29.tgz#5002e14f75e2d71e564281df0431c8c1b4a2a36a"
   integrity sha1-UALhT3Xi1x5WQoHfBDHIwbSio2o=
 
-"@types/minimist@^1.2.0", "@types/minimist@^1.2.2":
+"@types/minimist@^1.2.0":
```

Comment:
Question: Can this be combined with the one below?

Reply:
Who are you asking? 🙂

Reply:
Just screaming into the void lol

Reply:
You're free to push changes directly to this PR if you see optimizations that we should take